net/quagga/src/opnsense/mvc/app/controllers/OPNsense/Quagga/forms/dialogEditBGPRouteMaps.xml

```diff
@@ -0,0 +1,40 @@
+<form>
+  <field>
+    <id>routemap.enabled</id>
+    <label>Enabled</label>
+    <type>checkbox</type>
+    <help>Enable / Disable</help>
+  </field>
+  <field>
+    <id>routemap.name</id>
+    <label>Name</label>
+    <type>text</type>
+    <help>Route-map Name</help>
```

Comment:
does not really help
Maybe something like "The route map name must be unique and is used to hook your route map to after routing information is received or before it is sent.".